beginner_source/transfer_learning_tutorial.py

```diff
@@ -253,7 +253,9 @@ def visualize_model(model, num_images=6):
             images_so_far += 1
             ax = plt.subplot(num_images//2, 2, images_so_far)
             ax.axis('off')
-            ax.set_title('predicted: {}'.format(dset_classes[labels.data[j]]))
+            ax.set_title('predicted: {}, actual: {}'.format(
```

Comment:
Thanks for catching that! Showing both predicted and actual takes too much space. Can you please just show predicted?